.github/dependabot.yml

```diff
@@ -5,13 +5,10 @@ updates:
     schedule:
       interval: "daily"
   - package-ecosystem: "nuget"
-    directory: "/src/Extensions/Integration"
-    schedule:
-      interval: "daily"
-      
-  # This traces the dependencies through the dependency chain:
-  # Generator3, GirModel, GirLoader
-  - package-ecosystem: "nuget"
-    directory: "/src/Generation/GirTool"
+    directory:
```

Comment:
I think you used the wrong keyword. See https://docs.github.com/en/code-security/dependabot/working-with-dependabot/dependabot-options-reference#directories-or-directory--